### ISSUE TITLE
Empty the clone_url column on the external_service_repos table

### ISSUE THREAD

**Issue by asdine** [sourcegraph/sourcegraph#20124]
Once #20123 is done, we need to empty the `clone_url` column of the `external_service_repos` table, so that the tokens are no longer accessible.
After a release or two, we can drop that table.

**Comment by asdine** [comment]
Closing this in favor of https://github.com/sourcegraph/sourcegraph/pull/20247
